pkg/epp/metrics/metrics.go

```diff
@@ -31,16 +31,16 @@ import (
 )
 
 const (
-	InferenceModelComponent = "inference_model"
-	InferencePoolComponent  = "inference_pool"
-	InferenceExtension      = "inference_extension"
+	InferenceObjectiveComponent = "inference_objective"
```

Comment:
those metrics are model centric, I think we want to keep them as is, which we will continue to have model and target_model_name even after the migration to InferenceObjective.

Reply:
we likely want a similar set of metrics labeled by objective somehow.

Reply:
I think we should add a label, so metrics should have objective id, model name and resolved model name (all 3 labels)